change/@ni-nimble-components-6eb6eef1-eec2-4f6d-a98f-f7182ecdad50.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Any idea why the Chromatic status check isn't reporting that the new stories add baselines that require approval?

Reply:
Do you think we want/need Chromatic diffs for these stories?

Currently I opted them out of it (see [here](https://github.com/ni/nimble/blob/e070de537d61dfd326176d73e7f1070b39ca0c70/packages/nimble-components/src/label-provider/base/label-provider-stories-utils.ts#L53)), like some other documentation-related stories do. I should have commented on it to point it out and get feedback though.

Reply:
I don't feel too strongly but I think I'd vote for consistency with the other token stories (they don't opt out of Chromatic). Even though a change to these tokens isn't a visual diff from an app's point of view, as a reviewer I think I'd get some benefit from noticing that Chromatic has detected a change. Seeing those diffs might remind me to think about the change a bit more carefully.

Reply:
Re-opted into Chromatic snapshots